jekyll-serve.sh

```diff
@@ -6,4 +6,4 @@ bundle exec jekyll build
 bundle exec pandoc -s --from html _site/terms/word.html --reference-doc=assets/msword/styles.docx -o assets/msword/gruntwork-terms.docx
 echo -e "\e[1;32mA new Terms of Service document (/assets/msword/gruntwork-terms.docx) has been generated."
 echo -e "\e[1;31mRun Jekyll serve to watch for changes"
-bundle exec jekyll serve --drafts --incremental --host 0.0.0.0
+bundle exec jekyll serve --livereload --drafts --host 0.0.0.0
```

Comment:
Why not `--incremental`? Is that what breaks CSS reload?

Reply:
Yes, this is what broke the CSS reload. The `--incremental` flag is still experimental and only watches dependencies for `includes` and `layout` files.

https://jekyllrb.com/docs/configuration/incremental-regeneration/

Reply:
Gotcha. What impact does this have on hot reload performance?

Reply:
It increases hot reload by approximately 1 second.

Reply:
Hm, OK... We should keep our eye on this. As the amount of content goes up (which it will with the deployment guides!), this overhead will get worse without `--incremental`.